content/en/news/releases/1.6.x/announcing-1.6.10/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Announcing Istio 1.6.10
+linktitle: 1.6.10
+subtitle: Patch Release
+description: Istio 1.6.10 patch release.
+publishdate: 2020-09-22
+release: 1.6.10
+aliases:
+    - /news/announcing-1.6.10
+---
+
+This release contains bug fixes to improve robustness. This release note describes
+what’s different between Istio 1.6.9 and Istio 1.6.10.
+
+{{< relnote >}}
+
+## Changes
+
+- **Removed** deprecated help message for manifest migrate ([Istio #26230](https://github.com/istio/istio/issues/26230))
```

Comment:
Do we want this to say istioctl manifest migrate?

Reply:
fixed